_javascript/api_references.md

```diff
@@ -359,6 +359,9 @@ right_code: |
     });
     ```
 ---
+### Hint
```

Comment:
Can this be moved to the "Getting Started" section? It doesn't really fit in "API References", and it also throws off the alignment of the 'WebSocket' code in the right pannel with the 'WebSocket' description in the middle.

Also, instead of calling it "Hint" can the header read "Developer Console"

Reply:
Correct. I moved it into `Getting started` section. What's more, I created subsections for both `Installation` and `Developer Console` (as on `steemd_nodes.md`). Hope you like it.